README.md

```diff
@@ -45,14 +46,15 @@ brew install pandoc
 
 ```sh
 brew install R
+pip install -r requirements.txt # requirements have to be downloaded after R for some packages
```

Comment:
Same here, why do we have a requirement.txt file instead of using pyproject.toml?

Reply:
switched to use the regular poetry install